boskos-release/0.1/boskos-release.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: boskos-release
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "boskos,test"
+spec:
+  description: |
```

Comment:
It would be great if you could please add the description in the form of summary and description 

For e.g.

```yaml
spec:
     description: >-
       One line summary of the task.         👈 # Summary

       Description of the task 
  ```